### PR TITLE
Adding better rewrites, HEAD on feeds, broken link

### LINKED ISSUE #473: Support HEAD on /feed/rss
Apparently a bunch of feed readers "probe" with a HEAD rather than a GET. Today they'll get a 405 (method not supported)
image

### LINKED ISSUE #474: Support wfw:commentRss and fix broken links
We say (and we used to) support RSS of comments.

```
<wfw:comment>https://www.hanselman.com/blog/how-to-use-autocomplete-at-the-command-line-for-dotnet-git-winget-and-more/comments#comments-start</wfw:comment>
<wfw:commentRss>https://www.hanselman.com/blog/feed/rss/comments/969c3109-1daa-43d4-9913-450e44877aec</wfw:commentRss>
<slash:comments>10</slash:comments>
```

But 

`https://www.hanselman.com/SyndicationService.asmx/GetEntryCommentsRss?guid=c985231d-9b0e-4192-b3b0-871e207622d9 
`
should map to 

`https://www.hanselman.com/blog/feed/rss/comments/c985231d-9b0e-4192-b3b0-871e207622d9
`
and 

`https://www.hanselman.com/blog/feed/rss/comments/c985231d-9b0e-4192-b3b0-871e207622d9`

should return SOMETHING. Today it's empty.

- [x] Map old asmx endpoint to new one
- [ ] Fix /feed/rss/comments endpoint

### PR DESCRIPTION
**Pull request by shanselman**
This adds appinsights, closes #473 and starts work on #474. Unless we are just returning OK on comments, then it closes #474 as designed

### CODE REVIEW

File: source/DasBlog.Web.UI/DasBlog.Web.csproj [hunk 1]
```diff
@@ -13,9 +13,11 @@
     <Platforms>AnyCPU</Platforms>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <UserSecretsId>d3583964-0aca-4de4-9521-c74cdf42f990</UserSecretsId>
```
Comment (poppastring) [inline source/DasBlog.Web.UI/DasBlog.Web.csproj:16]: What is this UserSercretId?

File: source/DasBlog.Web.UI/Startup.cs [hunk 2]
```diff
@@ -288,7 +288,8 @@ public void ConfigureServices(IServiceCollection services)
 			{
 				options.WaitForJobsToComplete = true;
 			});
-		}
+            services.AddApplicationInsightsTelemetry(Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
+        }
```
Comment (poppastring) [inline source/DasBlog.Web.UI/Startup.cs:292]: Does this get stored in appsettings.json?

Comment (shanselman) [reply source/DasBlog.Web.UI/Startup.cs:292]: Yes, or in your AppService ENV variables. I believe it's optional.

File: source/DasBlog.Web.UI/DasBlog.Web.csproj (hunk 1 above)
Comment (shanselman) [reply source/DasBlog.Web.UI/DasBlog.Web.csproj:16]: I'm not sure, I believe this is added by AppServices...sorry, I'm struggling to keep my instance separate from the "generic" instance.

**Pull request merged**